homework_6_191122/src/Employee.java

```diff
@@ -34,10 +33,10 @@ public double getSalary() {
     }
 
     public double getRaiseSalary(double percent) {
-        return this.saldoSalary = percent / 100 * salary;
+        return percent / 100 * salary;
```

Comment:
this method should increase existing salary value:
1) Calculate increasing amount
2) Add it to current salary and reassign
3) Return increased value
double increaseAmount = percent / 100 * salary;
salary += increaseAmount;
return salary;

Reply:
Fixed. Took into account the comments above.